source/documentation/administration_guide/chap-Renewing_certificates.adoc

```diff
@@ -0,0 +1,36 @@
+:_content-type: ASSEMBLY
+[id="chap-Renewing_certificates_{context}"]
+= Renewing certificates before they expire
+
+[role="_abstract"]
+{virt-product-fullname} certificates now follow a standard 398 day lifetime and *must* be renewed once per year.
+
+[WARNING]
+====
+Do _NOT_ let certificates expire. If they expire, the environment becomes non-responsive and recovery is an error prone and time consuming process. If they do expire, you can manually renew them by following the instructions in link:https://access.redhat.com/solutions/3532921[How to manually renew RHV host SSL certificate if expired?]. Once you have completed the steps to manually renew the certificates, follow the steps in this procedure to properly enroll {virt-product-shortname} host certificates.
+====
+
+.Procedure
+
+. In the Administration Portal, click menu:Compute[Hosts].
+. Migrate the virtual machines away from the host.
+. Click menu:Management[Maintenance] and btn:[OK].
+. Once the host is in maintenance mode, click menu:Installation[Enroll Certificate].
+. Once enrollment is complete, click menu:Management[Activate].
+
+.Next steps
```

Comment:
This is a separate procedure. I'm not sure what "Next steps" is used for in RHV docs. I would probably call it "Renewing the Engine certificate" and add a more descriptive title to the procedure for renewing the host certificate.